Microsoft.Toolkit.Mvvm.SourceGenerators/Input/ICommandGenerator.cs

```diff
@@ -0,0 +1,341 @@
+                    IdentifierName(commandInterfaceTypeSymbol.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)),
+                    Identifier(propertyName))
+                .AddModifiers(Token(SyntaxKind.PublicKeyword))
+                .AddAttributeLists(
+                    AttributeList(SingletonSeparatedList(
+                        Attribute(IdentifierName("global::System.CodeDom.Compiler.GeneratedCode"))
+                        .AddArgumentListArguments(
+                            AttributeArgument(LiteralExpression(SyntaxKind.StringLiteralExpression, Literal(typeof(ICommandGenerator).FullName))),
+                            AttributeArgument(LiteralExpression(SyntaxKind.StringLiteralExpression, Literal(typeof(ICommandGenerator).Assembly.GetName().Version.ToString()))))))
+                    .WithOpenBracketToken(Token(summaryTrivia, SyntaxKind.OpenBracketToken, TriviaList())),
+                    AttributeList(SingletonSeparatedList(Attribute(IdentifierName("global::System.Diagnostics.DebuggerNonUserCode")))),
+                    AttributeList(SingletonSeparatedList(Attribute(IdentifierName("global::System.Diagnostics.CodeAnalysis.ExcludeFromCodeCoverage")))))
+                .WithExpressionBody(
+                    ArrowExpressionClause(
+                        AssignmentExpression(
+                            SyntaxKind.CoalesceAssignmentExpression,
+                            IdentifierName(fieldName),
+                            ObjectCreationExpression(IdentifierName(commandClassTypeSymbol.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)))
+                            .AddArgumentListArguments(Argument(
+                                ObjectCreationExpression(IdentifierName(delegateTypeSymbol.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)))
+                                .AddArgumentListArguments(Argument(IdentifierName(methodSymbol.Name))))))))
+                .WithSemicolonToken(Token(SyntaxKind.SemicolonToken));
+
+            return new MemberDeclarationSyntax[] { fieldDeclaration, propertyDeclaration };
+        }
+
+        /// <summary>
+        /// Get the generated field and property names for the input method.
+        /// </summary>
+        /// <param name="context">The input <see cref="GeneratorExecutionContext"/> instance to use.</param>
+        /// <param name="methodSymbol">The input <see cref="IMethodSymbol"/> instance to process.</param>
+        /// <returns>The generated field and property names for <paramref name="methodSymbol"/>.</returns>
+        [Pure]
+        private static (string FieldName, string PropertyName) GetGeneratedFieldAndPropertyNames(GeneratorExecutionContext context, IMethodSymbol methodSymbol)
+        {
+            string propertyName = methodSymbol.Name;
+
+            if (SymbolEqualityComparer.Default.Equals(methodSymbol.ReturnType, context.Compilation.GetTypeByMetadataName("System.Threading.Tasks.Task")!) &&
```

Comment:
Why a suppression is needed? Equals parameters are null annotated

Reply:
Ah, you're right, thanks! Fixed in https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3873/commits/e97a60b19fbaa61882995de5acd32a15a736ea98.
I think I had originally added that suppression due to missing annotations with some older preview version of the Roslyn NuGet packages, and tghen forgot to go back and remove that here after updating the references 😄